etc/helm/pachyderm/templates/enterprise-server/deployment.yaml

```diff
@@ -106,17 +106,50 @@ spec:
               name: {{ (include "pachyderm.enterpriseSecretSecretName" . ) | trim | quote }}
               key: "enterprise-secret"
         {{- end }}
-        {{- if or .Values.pachd.rootTokenSecretName .Values.pachd.rootToken }}
+        {{- if and (include "pachyderm.enterpriseLicenseKeySecretName" .) .Values.pachd.activateAuth }}
+        - name: ACTIVATE_AUTH
+          value: "true"
         - name: AUTH_ROOT_TOKEN
-          {{- if .Values.pachd.rootTokenSecretName }}
           valueFrom:
             secretKeyRef:
+            {{- if .Values.pachd.rootTokenSecretName }}
               name: {{ .Values.pachd.rootTokenSecretName | quote }}
+            {{- else }}
+              name: "pachyderm-auth"
+            {{- end }}
               key: "root-token"
-          {{- else }}
-          value: {{ .Values.pachd.rootToken | quote }}
-          {{- end }}
+        - name: AUTH_CONFIG
+          valueFrom:
+            secretKeyRef:
+              name: "pachyderm-auth"
+              key: "auth-config"
+        - name: IDP_CLIENTS
+          valueFrom:
+            configMapKeyRef:
+              name: "pachyderm-identity-config"
+              key: "oidc-clients"
+        {{- if or (include  "pachyderm.mockIDPRoleBindings" .) .Values.pachd.pachAuthClusterRoleBindings }}
+        - name: AUTH_CLUSTER_RBAC
+          valueFrom:
+            secretKeyRef:
+              name: "pachyderm-auth"
+              key: "cluster-role-bindings"
+        {{- end }}
+        {{- end }}
+        {{- if .Values.console.enabled }}
+        - name: CONSOLE_OAUTH_ID
```

Comment:
In the enterprise case, can users have multiple consoles?  How do we distinguish them now?  Do they all just use the same OAuth client?

Reply:
so this is kind of annoying but they're expected to be unique client IDs. We don't have a good way right now of preventing collisions besides customers setting the IDs explictly in the case of enterprise server with many clusters